docs/integrations/sources/github.md

```diff
@@ -86,6 +86,7 @@ Your token should have at least the `repo` scope. Depending on which streams you
 
 | Version | Date       | Pull Request | Subject |
 | :------ | :--------  | :-----       | :------ |
+| 0.1.8   | 2021-08-30 | [5726](https://github.com/airbytehq/airbyte/pull/5726) | Use `airbytehq/integration-test` repo for tests instead of `airbytehq/airbyte` |
```

Comment:
we don't really need a version bump here since we're just updating tests

Reply:
Ok. Will remove it.